include/oneapi/dpl/version

```diff
@@ -0,0 +1,34 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef __ONEDPL_version_H
+#define __ONEDPL_version_H
+
+#include "oneapi/dpl/internal/common_config.h"
```

Comment:
This is a logic change. onedpl_config had never have includes. Now it includes common_config.h and furthermore, `<version>`

I am not sure if it's critical or not. Need to double check

Reply:
As of today, the config file already includes `<version>`, so I guess it's not an issue.